src/main/java/org/apache/sling/pipes/PipeBuilder.java

```diff
@@ -345,4 +345,11 @@
      * @throws PersistenceException in case something goes wrong in the job creation
      */
     Job runAsync(Map bindings) throws PersistenceException;
+
+    /**
+     * run referenced pipes in parallel
+     * @param numThreads number of threads to use for running the contained pipes
+     * @return set of resource path, merged output of pipes execution (order is arbitrary)
+     */
+    ExecutionResult runParallel(int numThreads) throws Exception;
```

Comment:
i guess i would have prefer execute or something , calling that method with `1` thread is very legit and will be used (you just want a series of N pipes to be executed sequentially, with one call)